pkg/apis/command/types.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2024 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	// ResourceKindReschedule is kind name of Reschedule.
+	ResourceKindReschedule = "Reschedule"
+	// ResourceSingularReschedule is singular name of Reschedule.
+	ResourceSingularReschedule = "reschedule"
+	// ResourcePluralReschedule is plural name of Reschedule.
+	ResourcePluralReschedule = "reschedules"
+	// ResourceNamespaceScopedReschedule indicates if Reschedule is NamespaceScoped.
+	ResourceNamespaceScopedReschedule = false
```

Comment:
Can the resource be set to a namespace scope so that the permission control can be more fine-grained?

Reply:
this api can select a non-namespacescoped resource like clusterrole, so it may not suitable to be namespacescoped.